src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs

```diff
@@ -249,13 +249,14 @@ public bool IsUnityProject()
             var globalProperties = new Dictionary<string, string>
             {
                 { PropertyNames.DesignTimeBuild, "true" },
+                { PropertyNames.BuildingInsideVisualStudio, "true" },
```

Comment:
😆 I don't see any way to avoid this one.

Reply:
Yeah, the property name is kind of a misnomer. Originally, it was added for design-time scenarios like code generators in VS.